### DISCUSSION TITLE
New feature: add maps downloading script into release

### DISCUSSION THREAD

**Discussion by fheroes2bugs** [ihhub/fheroes2#8849] [Ideas]
### Preliminary checks

- [X] I've checked that there aren't [**other open issues**](https://github.com/ihhub/fheroes2/issues?q=is%3Aissue) on the same topic.

### Describe the problem requiring a solution

I can see some gamers already started recreating maps using new editor:
https://www.youtube.com/watch?v=OMBvU3W7qzY

It would be great if we could collect all the maps, that are done with editor to somewhere and just add it to release or add script to release, that downloads them automatically.

### Describe the possible solution

Add maps download script for release. It would prevent users from necessity of finding where to download original game or maps. You just download beta + add maps there, that were created by new editor.

### Additional info

Jon Van Caneghem, GOG and Ubisoft don't care about heroes2 terms of use. We could also just add link to all the maps (not only created with editor). 28 years passed since game was published. Link to download is not prohibited.
3DO and NWC are dead.

**Comment by ihhub** [comment]
First of all, you must own a full original game. If I understand properly you meant "beta" as a demo version. fh2m maps don't work on a demo version.

> Jon Van Caneghem, GOG and Ubisoft don't care about heroes2 terms of use. We could also just add link to all the maps (not only created with editor). 28 years passed since game was published. Link to download is not prohibited.
3DO and NWC are dead.

Where is the basis of these statements? Please read how copyright laws work.

**Reply by oleg-derevenetz** [reply]
> fh2m maps don't work on a demo version.

Just for clarification: fh2m maps (and editor) will only work if you have the Price of Loyalty version of the game. Having the Succession Wars version (even the "full/official" one) is not enough.

**Comment by ihhub** [comment]
Having any extra application: launcher, script and etc, is also not a proper way to make things right. `fheroes2` engine will come with a set of maps but these maps **must** be approved and verified.

I am moving this topic into the discussion section.

**Reply by fheroes2bugs** [reply]
> `fheroes2` engine will come with a set of maps but these maps **must** be approved and verified

I suggest to add this "set of maps" to roadmap.
